bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/database/migration/Migration2to3.kt

```diff
@@ -0,0 +1,19 @@
+package io.bucketeer.sdk.android.internal.database.migration
+
+import android.content.SharedPreferences
+import androidx.sqlite.db.SupportSQLiteDatabase
+import io.bucketeer.sdk.android.internal.Constants
+import io.bucketeer.sdk.android.internal.evaluation.db.EvaluationEntity
+import io.bucketeer.sdk.android.internal.event.EventEntity
+
+class Migration2to3 : Migration {
+
+  override fun migrate(db: SupportSQLiteDatabase, sharedPreferences: SharedPreferences) {
+    db.delete(EvaluationEntity.TABLE_NAME, null, null)
```

Comment:
Can you add a comment that describes why we need to delete data in Evaluation table, so that we can better understand the intention in the future?

Reply:
@yshrsmz I have added a comment.
https://github.com/bucketeer-io/android-client-sdk/pull/79/files#diff-439703daa4fc7dfb6c064763b3f3a622dd26367a1e749131e5195e8ab3e5c6f1R12-R14

Thanks!